CoffeeChat/Controller/EditingViewController.swift

```diff
@@ -82,36 +82,61 @@ class EditingViewController: UIViewController {
     private let tableView = UITableView(frame: .zero, style: .grouped)
 
     // MARK: - Display Settings
-    var showsGroups = true
-    private var showingLess = true
-    private var hideAfter = 3 // Doesn't display more [hideAfter] categories if showingLess is true
+    private var showsGroups = true
+    private var showingLessRows = true
+    private var numberHiddenShownRows = 3 // Displays only [numberHiddenShownRows] if showingLessRows is true
 
     private var sections: [Section] = []
+
     // More Section and count
     private var moreSection: Section? {
         get {
             if let loc = sections.firstIndex(where: { $0.type == .more }) {
                 return sections[loc]
-            } else { return nil }
+            } else {
+                return nil
+            }
         }
     }
-    private var moreCount: Int {
-        get {
-            return moreSection?.filteredItems.count ?? 0
-        }
+    private var moreSectionSize: Int {
+        get { return moreSection?.filteredItems.count ?? 0 }
     }
+
     // Your section and count
     private var yourSection: Section? {
         get {
             if let loc = sections.firstIndex(where: { $0.type == .yours }) {
                 return sections[loc]
-            } else { return nil }
+            } else {
+                return nil
+            }
         }
     }
-    private var yourCount: Int {
-        get {
-            return yourSection?.filteredItems.count ?? 0
-        }
+
+    private var yourSectionSize: Int {
+        get { return yourSection?.filteredItems.count ?? 0 }
+    }
+
+
+    // Initialization
+    private init() {
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    static func createForInterests() -> EditingViewController {
+        let editingVC = EditingViewController()
+        editingVC.showsGroups = false
+        return editingVC
+    }
+
+    static func createForGroups() -> EditingViewController {
+        let editingVC = EditingViewController()
+        editingVC.showsGroups = true
+        return editingVC
```

Comment:
These are so similar that maybe it would make more sense to take `showsGroups` as a parameter.